### DISCUSSION TITLE
이름이 왜 곧감인가요?

### DISCUSSION THREAD

**Discussion by KimGunwWoo** [daon1213/Goj_Gam#4] [Q&A] [answered]
불편하네요;;

**Comment by daon1213** [comment, marked as answer]
배달 앱 중에 무언가 독창성 있는 이름이 없을까 고민하던 도중 여러 친구들에게 물어봤는데 어떤 친구가 "곶감" 이라고 하면서 "곧감" 즉 '빨리 간다' 라는 말의 응용이다. 라고 얘기하였는데 순간 그게 맘에 들어서 결정하게 되었습니다 ㅎ
